WebUI/src/SourceCodeView.css

```diff
@@ -0,0 +1,70 @@
+/*
+  Copyright (c) 2020 The Orbit Authors. All rights reserved.
+  Use of this source code is governed by a BSD-style license that can be
+  found in the LICENSE file.
+*/
+
+:root {
+    --feature-color: #19181a;
+}
+
+html,
+body {
+    margin: 0;
+    padding: 0;
+    border: 0;
+    background-color: #2d2d2d;
+    font-family: Arial, Helvetica, sans-serif;
```

Comment:
can we ensure, to use the same font (and size) as in the rest of orbit?

Reply:
on it. there is more that needs to be done. The button and the scroll bar for example.

Reply:
do you want to address this in this PR?

Reply:
Yes. The are still quite some (minor) changes coming based on the feedback on Monday. I will mark it ready for review when I'm done.

Reply:
The style is now adjusted to look more or less like Orbit.
I'm a bit hesitant to make the font dynamicly adjusting since this
might affect the layout and result in a bigger problem than a font
that is a little bit off.

Reply:
okay. thanks anyways.